web/src/components/UpcomingEventsList.tsx

```diff
@@ -0,0 +1,28 @@
+import SimpleSlider from "./SimpleSlider";
+import { Event } from "../types/types";
+import UpcomingEventCard from "./UpcomingEventCard";
+import { Link } from "react-router-dom";
+
+interface UpcomingEventsListProps {
+  upcomingEvents: Event[];
+}
+
+const UpcomingEventsList: React.FC<UpcomingEventsListProps> = ({
+  upcomingEvents,
+}) => {
+  return (
+    <>
+      <SimpleSlider cardType="upcoming">
+        {upcomingEvents.map((event, index) => (
+          <div key={index} className="p-2 hover:cursor-pointer">
```

Comment:
using index as key is not the best. I think events has an ID. if so please use that. if there isn't an ID. then it should be fine

Reply:
Yeah good shout, I'll change it to the event.id!